onnxruntime/python/tools/transformers/models/t5/t5_helper.py

```diff
@@ -151,21 +151,18 @@ def export_onnx(
     def auto_mixed_precision(
         onnx_model: OnnxModel,
         op_block_list: List[str] = [
-            "Pow",
-            "ReduceMean",
-            "Add",
-            "Sqrt",
-            "Div",
+            "SimplifiedLayerNormalization",
+            "SkipSimplifiedLayerNormalization",
             "Mul",
```

Comment:
I think we can remove Mul (which was for mask filter value).

Reply:
removed